deltacat/utils/pyarrow.py

```diff
@@ -211,6 +211,26 @@ def _get_kwargs(self, content_type: str, kwargs: Dict[str, Any]) -> Dict[str, An
         return kwargs
 
 
+def _create_s3_file_system(s3_client_kwargs: dict) -> s3fs.S3FileSystem:
```

Comment:
Is this a PyArrow specific utility method?

Reply:
No, we can move this to s3fs util.